src/gui/editors/PianoRoll.cpp

```diff
@@ -2813,16 +2753,13 @@ void PianoRoll::dragNotes( int x, int y, bool alt, bool shift, bool ctrl )
 		else
 		{
 			// shift is not pressed; stretch length of selected notes but not their position
-			for (Note *note : notes)
+			for (Note *note : getSelectedNotes())
```

Comment:
I noticed this as well but let it be since getSelectedNotes would only be called once either way, and I was wary of accidentally reusing selectedNotes for too long. However, I think you're right about this since the selection isn't modified in this function.